e2e/nomostest/nt.go

```diff
@@ -579,9 +579,6 @@ func (nt *NT) WaitForRepoSyncs(options ...WaitForRepoSyncsOption) {
 	syncTimeout := waitForRepoSyncsOptions.timeout
 
 	if nt.MultiRepo {
-		if err := ValidateMultiRepoDeployments(nt); err != nil {
-			nt.T.Fatal(err)
-		}
```

Comment:
The validation here guarantees no Pod is in a CrashLoopBackoff state because of OOMKilled.

Reply:
ACKed. Skipping the valiation here may make the tests more flaky.
I am trying to see how much time skipping this validation here can save, and how much worse it makes the flakiness. 
Mostly likely we won't need this change after https://github.com/GoogleContainerTools/kpt-config-sync/pull/116 is merged.